aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -2078,26 +2078,35 @@ std::tuple<Tensor,Tensor> _linalg_qr_helper_cuda(const Tensor& self, std::string
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ symeig ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Does this comment need to be updated?

Reply:
Yes, probably. TBH I don't get why they're needed 😄